### DISCUSSION TITLE
Avoid use of JDK8Module

### DISCUSSION THREAD

**Discussion by mebigfatguy** [graphql-java-kickstart/graphql-java-servlet#570] [Q&A] [answered]
My understanding (please correct me) is that jackson-datatype-jdk8 is no longer necessary as it's been merged to other jars. However, when using graphql, i still see a 

 mapper.registerModule(new Jdk8Module());

at
 
java.lang.ClassNotFoundException: com.fasterxml.jackson.datatype.jdk8.Jdk8Module
Caused by:java.lang.NoClassDefFoundError: com/fasterxml/jackson/datatype/jdk8/Jdk8Module  at graphql.kickstart.execution.config.ConfiguringObjectMapperProvider.&lt;init&gt;(ConfiguringObjectMapperProvider.java:17) 

Is there a way to tell graphql, not to register that module, or am i confused?

**Comment by federicorispo** [comment]
@mebigfatguy Hi, the `graphql-java` library now supports jdk-11 or above so the Jdk8Module for Jackson is no longer needed! I will open a PR asap to fix this!

Thanks for your feedback!

**Comment by federicorispo** [comment, marked as answer]
Fixed: https://github.com/graphql-java-kickstart/graphql-java-servlet/pull/574
